### PR TITLE
Add mission and purpose section to readme

### PR DESCRIPTION
**Pull request by kubi-dev**
To remind us about our long-term vision and to share this vision with others

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 # Charity Token
 
-The Charity Token is an Ethereum-based, ERC20 compliant token issued for collecting funds for charities. The initial goal of the project is to give donors an possibility to transfer Ether for one closer specified organisation and issue tokens in return. In a later stage, the issued Charity Token will be used to choose on charity organisations that should be supported.
+The Charity Token is an Ethereum-based, ERC20 compliant token issued for collecting funds for charities. The initial goal of the project is to give donors a possibility to transfer Ether for one closer specified organisation and issue tokens in return. In a later stage, the issued Charity Token will be used to choose on charity organisations that should be supported.
+
+## Mission and purpose
+
+The Charity Token is inspired by The Pineapple Fund which is a philanthropic project. An anonymous individual founded it in December 2017 in order to give away 5,057 Bitcoins to charity.
+
+Many people in crypto industry got rich over the recent years. We think many of them would like to share their prosperity with those in need.
```
Comment (michald) [inline README.md]: > the crypto industry

File: README.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
 # Charity Token
 
-The Charity Token is an Ethereum-based, ERC20 compliant token issued for collecting funds for charities. The initial goal of the project is to give donors an possibility to transfer Ether for one closer specified organisation and issue tokens in return. In a later stage, the issued Charity Token will be used to choose on charity organisations that should be supported.
+The Charity Token is an Ethereum-based, ERC20 compliant token issued for collecting funds for charities. The initial goal of the project is to give donors a possibility to transfer Ether for one closer specified organisation and issue tokens in return. In a later stage, the issued Charity Token will be used to choose on charity organisations that should be supported.
+
+## Mission and purpose
+
+The Charity Token is inspired by The Pineapple Fund which is a philanthropic project. An anonymous individual founded it in December 2017 in order to give away 5,057 Bitcoins to charity.
```
Comment (michald) [inline README.md]: I'd get rid of the semicolon here `5,057` to make it clear you mean 5 thousand.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+Many people in crypto industry got rich over the recent years. We think many of them would like to share their prosperity with those in need.
+
+Our goal is to create an easy and trustless crypto-based way to donate charities. We want to create a solution that will be of use anytime someone in crypto space feels like to make an impact.
```
Comment (michald) [inline README.md]: > We want to create a solution that can be used anytime someone in the crypto space feels that wants to make an impact.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+
+Charity Token is an universal way to do something good to others. We don't want to make donators think which organisation they are supposed to support. We want to make them sure that their assets are  spent for a good cause and make a positive impact on our society.
```
Comment (michald) [inline README.md]: 1. `a universal`
2. the redundant whitespace between `assets are` and `spent`.

File: README.md [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
+
+Our final vision is to create an  always on-going ICO for Charity Token. The token will be supported by a decentralized governance mechanism. It will allow token holders to decide which organisation should be supported with  collected fund. We invite entire crypto community to a discussion upon this governance mechanism and designing its framework.
```
Comment (michald) [inline README.md]: 1. the redundant whitespace between `create an` and `always on-going`.
2. the redundant whitespace between `supported with` and `collected fund`.
3. `collected funds`, not `collected fund`
4. > We invite the entire crypto community to a discussion on the governance mechanism and designing its framework.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,20 @@
...
+
+Our first iteration (currently  in progress) supports just one organisation chosen by us. We decided to begin  with this baby step to show it is not mere  talking, but actions. Here we would like to  to start a discussion about a final version of the governance protocol.
```
Comment (michald) [inline README.md]: Redundant whitespaces + grammar mistakes, I've corrected them:
> Our first iteration (currently in progress) supports just one organisation chosen by us. We decided to begin with this baby step to show it is not merely talking, but actions. Here we would like to start a discussion about a final version of the governance protocol.

File: README.md (hunk 1 above)
Comment (PiotrMisiurek) [reply README.md]: 👍

File: README.md (hunk 4 above)
Comment (PiotrMisiurek) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (PiotrMisiurek) [reply README.md]: Both are correct, I stay with the first one

**Review by michald — APPROVED** [review]

**Pull request merged**